### PR TITLE
Multilingual

### PR DESCRIPTION
**Pull request by line-o**

### CODE REVIEW

File: src/js/modules/info.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ function getPublicationDate(rawDate) {
     return '';
   }
   var date = new Date(rawDate);
-  return '<p>Veröffentlicht am: ' + date.getDate() + '.' + (date.getMonth() + 1) + '.' + date.getFullYear() + '</p>';
+  return '<p>' + $.i18n( 'tab_info_title_published' ) + ': ' + date.getDate() + '.' + (date.getMonth() + 1) + '.' + date.getFullYear() + '</p>';
```
Comment (plutonik-a) [inline src/js/modules/info.js]: Can we abstract the date format into variables here for different locales? Like i.e. for English `month, dd, yyyy` or `mm/dd/yyyy`?

**Comment by plutonik-a** [comment]
I tried to solve the date format issue in the "Date of Publication" (Info Tab) with [`toLocaleString()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleString), but I'm not sure, if I used the options correctly. It seems, that they override the locale format.
The aim of adding options was to delete the automated time output from the date.
Maybe there is a better way to prevent the method from displaying the time.

**Comment by Kambfhase** [comment]
> It seems, that they override the locale format.

Looks fine to me. Could you please elaborate by providing an expected result and the actual result.

**Comment by line-o** [comment]
Date formatting seems to be a show-stopper in regards of wikpedia/jquery-i18n.
We need to re-evaluate other options like writing/finding a plugin or using a different library like globalize.

**Comment by plutonik-a** [comment]
> Date formatting seems to be a show-stopper in regards of wikpedia/jquery-i18n.

Could you provide to a source for this? How would jquery-i18n mess up the processing of  `toLocaleString()`?

**Comment by line-o** [comment]
It does not mess with it. It does absolutely nothing with dates. That is the problem. We need something that provides date formatting (accross browsers and easy to use).

**Comment by alexander-heimbuch** [comment]
This repository is dedicated to the v4 web player. I will close this PR for this repository, please refer to https://github.com/podlove/podlove-web-player-v3

**Pull request closed without merging**